src/redux/sagas/utils/getGasPrices.ts

```diff
@@ -176,16 +183,44 @@ const fetchGasPrices = async (): Promise<GasPricesProps> => {
         suggested: cheaper.mul(150).div(100), // 50% more than network
         faster: cheaper.mul(200).div(100), // 100% more than network
 
-        maxFeePerGas,
-        maxPriorityFeePerGas,
+        maxFeePerGas: maxFeePerGas.add(maxFeePerGas.mul(5).div(100)), // 5% more
```

Comment:
Would it make sense to extract the 5% into a constant given we use it in multiple places and then just multiply with that value?

Reply:
Kinda awkward for a constant, could be done with a helper function though